snuba/views.py

```diff
@@ -308,6 +308,9 @@ def parse_and_run_query(dataset, request: Request, timer) -> QueryResult:
             list(filter(lambda cond: cond not in prewhere_conditions, request.query.get_conditions()))
         )
 
+    for processor in dataset.get_query_processors():
```

Comment:
potentially the PRE-WHERE processing may later become a default query processor as well.

Reply:
It would be great if sentry could no longer need to know which conditions should go into the prewhere, and instead send a simple collection of conditions.